CMakeLists.txt

```diff
@@ -67,11 +74,26 @@ if(${MYSOFA_FOUND})
     include_directories(${MYSOFA_INCLUDE_DIRS})
 endif(${MYSOFA_FOUND})
 
-add_library(spatialaudio STATIC ${sources})
+if(BUILD_STATIC_LIBS)
+    add_library(spatialaudio-static STATIC ${sources})
+    if(${MYSOFA_FOUND})
+        target_link_libraries(spatialaudio-static ${MYSOFA_LIBRARIES})
```

Comment:
Is this required? I'd expect sources only to affect the resulting static library (granted, my cmake skillz are... let's say rusted)